composer.json

```
@@ -17,7 +17,7 @@
   "require": {
     "php": ">=7.1",
     "ext-mbstring": "*",
-    "symfony/dependency-injection": "^4.2",
+    "symfony/dependency-injection": "^5.1",
```

Voor backwards compatibility kan je evt het volgende gebruiken omdat er verder niets breaking is
```suggestion
    "symfony/dependency-injection": "^4.2||^5.1",
```

of 

```suggestion
    "symfony/dependency-injection": "^4.0||^5.0",
```

---

Ik weet niet of dat in dit geval nuttig is.

Gebruik je Symfony 4 krijg je autofactory 1.1.x
Gebruik je Symfony 5 krijg je autofactory 1.2.x

`||` lijkt me bedoeld 

1. Als er 2 verschillende libraries zijn die allebei `symfony/dependency-injection` willen en dat je dan composer laat uitzoeken of je v4 of v5 krijgt. Echter  `symfony/dependency-injection` komt binnen via `symfony/framework-bundle` en daarmee staat de versie vast. 

2. Als we ook nog nieuwe features in autofactory beschikbaar willen maken voor Symfony 4. Echter dan moeten we deze nieuwe features ook testen op 2 versies.

---

Ik ging uit van het tweede geval maar als je geen nieuwe features verwacht die ook in projecten met Symfony 4 nodig zijn is dit ook prima inderdaad.
Mocht er nieuwe functionaliteit of fixes nodig zijn voor 4.x dan kan dit alsnog